### PR TITLE
Restore compatibility of mmapi::api_call

### PR DESCRIPTION
**Pull request by Martchus**
* Restore return value of mmapi::api_call which has changed in
  3655e0c1383457c49e8b4669c37624b23fb1c240.
* See https://progress.opensuse.org/issues/80412

### CODE REVIEW

File: mmapi.pm [hunk 1]
```diff
@@ -92,6 +92,8 @@ sub api_call {
     return ($res, $tx);
 }
 
+sub api_call { api_call_2(@_)->res }
```
Comment (okurz) [inline mmapi.pm:95]: please add a method description here and above.

**Review by okurz — CHANGES_REQUESTED** [review]
Please add method descriptions

Also I suggest you roll this change out temporarily on a machine, e.g. staging and run real openQA tests, e.g. clone https://openqa.opensuse.org/tests/1484464 and maybe also a real multi-machine test

File: mmapi.pm (hunk 1 above)
Comment (kalikiana) [reply mmapi.pm:107]: Ah, nice, I was just about to ask what the docs now state 👍

**Review by kalikiana — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Comment by Martchus** [comment]
I've cloned the job with the failing module https://openqa.opensuse.org/tests/1484451#step/yast2_nfs_server/60 and it works now locally.

**Comment by Martchus** [comment]
I would actually like to merge this before we deploy the broken version again. However, Travis didn't even enqueue the job. Hence I'm running the testsuite locally now and will merge the PR manually if it passes.

**Pull request merged**